File: Java_Programs/Demo2.java
// program to print 5 times "Marvellous" on screen 

class Demo2 
{
	public static void main(String arg[])
	{
		Demo dobj = new Demo();
		dobj.Display();
	}
}

class Demo 
{
	public void Display()
	{
		int i = 0;
		for(i = 1; i <= 5;i++)
		{
			System.out.println("Marvellous :" +i);
		}
	}
}

File: C++_Programs/Demo2.cpp
// program to print 5 times marvellous on screen using oops

#include<iostream>
using namespace std;

class Demo 
{
public:

	void Display()
	{
		int i = 0;
		for(i = 1; i <= 5; i++)
		{
			cout<<"Marvellous:"<< <<i<<"\n";
		}
	}
};

int main()
{
	Demo dobj;
	dobj.Display();

	return 0;
}